src/library_glfw.js

```diff
@@ -579,6 +579,11 @@ var LibraryGLFW = {
       var win = GLFW.WindowFromId(winid);
       if (!win) return;
       win.windowSizeFunc = cbfun;
+     
+#if USE_GLFW == 2
```

Comment:
Can you add a comment here explicitly describing this unintuitive behavioral difference between GLFW 2 and 3, and perhaps a link?